README.md

```diff
@@ -108,19 +108,10 @@ List current environment version for each microservice hosted in Kubernetes.
 #### Resources
 `./resources/cosmos-upload`  
 
-Upload multiple items to a local emulator instance of cosmos. Follow the command prompt to provide:
-- crn
-- sbi
-- database name
-- container name
-- quantity
-
-Currently the items uploaded match the messages-database schema only.
+Upload multiple items to a local emulator instance of Cosmos. Insert information as prompted. Currently the items uploaded match only the messages and queries schemas.
```

Comment:
This is fine for now, the upload script guides the user through the required fields so no need to elaborate it for now. Plus while still in alpha the fields may change frequently. Plus its only a spike